libraries/Adapters/Microsoft.Bot.Builder.Adapters.WeChat.Test/Microsoft.Bot.Builder.Adapters.WeChat.Test.csproj

```diff
@@ -1,27 +1,27 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Sorry for misunderstanding, will do it.

Reply:
thanks, I see you moved the project already, please also rename the project and the folder where the project is to Microsoft.Bot.Builder.Adapters.WeChat.Tests (plural) instead of Microsoft.Bot.Builder.Adapters.WeChat.Test (singular).